src/App.css

```diff
@@ -1,3 +1,260 @@
-*body {
-  background-color: white;
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+#root {
+  margin: 5% 4%;
+  display: flex;
+  background-color: #fafafa;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.app {
+  width: 100%;
+}
+
+.app nav {
+  width: 100%;
+  background: white;
+  border-bottom: 1px solid #dfdfdf;
+  height: 85px;
+  margin-top: -14px;
+}
+
+.app ul {
+  display: flex;
+  gap: 32px;
+  list-style-type: none;
+  align-items: center;
+  padding: 0 2%;
+}
+
+.img {
+  display: flex;
+  justify-content: flex-end;
+  width: 64%;
+}
+
```

Comment:
- [ ] Please add the `activeclass` effect on your `book` navlink as in the Zeplin design you can see that the book `navlink` is `#121212` color and the categories link is the same color `#121212` but with opacity `0.5` to make it look lighter in color . [OPTIONAL] You can use `activeclassname` to add this effect.
- [ ] Also please add a bit more gap between the logo and the `navlinks`.

Your Implementation
---
<img width="1341" alt="Screenshot 2023-07-29 at 10 06 03 AM" src="https://github.com/kerlos-bahaa/Bookstore-react/assets/44798044/06ca6353-7d9e-42c7-8c82-4d1944e71286">


Provided Zeplin design
------
<img width="981" alt="Screenshot 2023-07-29 at 9 54 34 AM" src="https://github.com/kerlos-bahaa/Bookstore-react/assets/44798044/16a6bb95-33a7-4ca5-a18c-852c6025ac0c">